dapr/proto/runtime/v1/appcallback.proto

```diff
@@ -25,25 +25,22 @@ service AppCallback {
   // Lists all topics subscribed by this app.
   rpc ListTopicSubscriptions(google.protobuf.Empty) returns (ListTopicSubscriptionsResponse) {}
 
-  // Listens events from Pubsub
+  // Subscribes events from Pubsub
   rpc OnTopicEvent(TopicEventRequest) returns (google.protobuf.Empty) {}
 
   // Lists all input bindings subscribed by this app.
-  rpc ListBindingsSubscriptions(google.protobuf.Empty) returns (ListBindingsSubscriptionsResponse) {}
+  rpc ListInputBindings(google.protobuf.Empty) returns (ListInputBindingsResponse) {}
```

Comment:
@yaron2 Please review this naming. Using `InputBindings`  explicitly seems more readable than `BindingsSubscriptions`. if you have any objections for this, I will revert it. 

cc/ @amanbha

Reply:
I'm fine with this change.